lib/Logger/LogMacros.h

```diff
@@ -106,6 +106,13 @@
                                   << arangodb::Logger::FILE(__FILE__)    \
                                   << arangodb::Logger::FUNCTION(__FUNCTION__))
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief logs a message for debugging during development
+////////////////////////////////////////////////////////////////////////////////
+
+#define LOG_DEVEL LOG_TOPIC(ERR,Logger::FIXME) << "###### "
```

Comment:
Why not use `Logger::DEVEL` as the topic here?
I am a bit worried that these macros will make us add even more log garbage than we already have. How can we prevent that from happening?

Reply:
I had devel first as log topic but it is unfortunately not visible in the default configuration.

Reply:
I have added this marco in the hope that I will be able to remove all my logging messages. Without forgetting any of them. The Logger::FIXME is all over the place and it is more writing than LOG_DEVEL. When a feature is merged back into devel the definition should be the only remaining LOG_DEVEL in the code base.